docs/howto/operate/move-hub.md

```diff
@@ -106,6 +110,53 @@ do your modifications. This prevents needing to create another EC2
 instance just for this.
 ```
 
+### Azure Files
+
+```{note}
+The exception: even if the source and target cluster are in the same Azure subscription, re-using the same home directory storage might prove more difficult than copying files from one storage to the other.
+```
+
+We also use AzureFiles as in-cluster NFS storage. These are the steps to transfer the home directories between a NFS server located in another cluster, but in the same Azure subscription with an AzureFile NFS storage.
+
+AzureFile needs to be mounted in the source NFS VM in order to copy the data.
+
+1. Make sure AzureFile has access to the VNet in which the source NFS VM is in. Documentation on how to do that is can be found [here](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-networking-endpoints?tabs=azure-portal#restrict-public-endpoint-access).
+
+2. On the source NFS VM:
+   1. Locate the private ssh key counterpart of the public one set for this cluster. The private key is usually encrypted with sops and stored into the repository where the infrastructure config of the cluster resides.
+      ```bash
+      sops -d secrets/ssh-key > secrets/ssh-key.unsafe
+      ```
+   2. Make sure `kubectl` is set to talk to the Azure cluster, that host the source VM, as we'll be 'hopping' through it to access the NFS VM.
+      ```bash
+      kubectl config use-context <the-desired-context>
```

Comment:
You may wish to add a note here that suggests using `kubectl config get-contexts` to list which contexts are available. Also, if the desired context isn't available, you can authenticate using `az aks get-credentials --name CLUSTER_NAME --resource-group RESOURCE_GROUP_NAME`